.travis.yml

```diff
@@ -42,7 +42,8 @@ notifications:
     - joshua.suereth@typesafe.com
 
 script:
-  - sbt -J-XX:ReservedCodeCacheSize=128m "$SCRIPTED_TEST"
+  - sbt ";publishLocal;project bundledLauncherProj;publishLocal;project precompiled-2_9_2;publishLocal;project precompiled-2_8_2;publishLocal;project precompiled-2_9_3;publishLocal"     
+  - sbt -J-XX:ReservedCodeCacheSize=128m -Dsbt.version=0.13.9-SNAPSHOT "$SCRIPTED_TEST"
```

Comment:
@eed3si9n Scripted is borked in some way.  you can't change its code in the current repoa nd see those changes.  I suggest, maybe, we just remove it from the codebase and modularize it.